projects/storefrontlib/src/lib/cms-lib/banner/responsive-banner.component.ts

```diff
@@ -11,42 +9,7 @@ import { Observable } from 'rxjs';
   changeDetection: ChangeDetectionStrategy.OnPush
 })
 export class ResponsiveBannerComponent extends BannerComponent {
-  // TODO: move to a more generic location
-  // TODO: Make configurable
-  private formats: any[] = [
-    { code: 'mobile', width: 200 },
-    { code: 'tablet', width: 500 },
-    { code: 'desktop', width: 800 },
-    { code: 'widescreen', width: 1200 }
-  ];
-
   getClass(): string {
-    return 'responsive-banner ' + this.component.uid;
-  }
-
-  getImageUrl(): Observable<string> {
-    return this.component.data$.pipe(
-      map(data =>
-        BannerComponent.hasMedia(data)
-          ? (<CmsResponsiveBannerComponentMedia>data.media).desktop.url
-          : ''
-      )
-    );
-  }
-
-  getResponsiveSrcSet(): Observable<string> {
-    return this.component.data$.pipe(
-      map(data => {
-        return this.formats.reduce((srcset, format) => {
-          if (typeof data.media[format.code] !== 'undefined') {
-            return (srcset += `${this.getBaseUrl()}${
-              data.media[format.code].url
-            } ${format.width}w, `);
-          } else {
-            return srcset;
-          }
-        }, '');
-      })
-    );
+    return 'responsive-banner ' + this.service.getComponentUID();
```

Comment:
Also can we assign 'responsive-banner' to some `const RESPONSIVE_BANNER_CLASS` or something like this to describe little bit what that string means. In addition in a future we can use that variable into some other methods if needed.